package.json

```diff
@@ -118,22 +118,22 @@
     "gulp-rename": "^1.4.0",
     "gulp-svgmin": "^1.2.3",
     "gulp-theo": "^2.0.0",
-    "husky": "^1.1.2",
+    "husky": "^1.2.0",
     "jest": "^21.2.1",
     "js-yaml": "^3.12.0",
     "json-loader": "^0.5.4",
     "lerna": "^3.4.3",
-    "lint-staged": "^8.0.4",
+    "lint-staged": "^8.0.5",
     "lodash": "^4.17.11",
-    "marked": "^0.5.1",
+    "marked": "^0.5.2",
     "merge-stream": "^1.0.1",
-    "node-sass": "^4.9.3",
+    "node-sass": "^4.10.0",
```

Comment:
The noisy deprecation warning for importing css is gone now. See https://github.com/sass/node-sass/releases/tag/v4.11.0.